### PR TITLE
first version

### PR DESCRIPTION
**Pull request by jgfidelis**
Screen Shot 2019-05-25 at 5 41 58 PM
Screen Shot 2019-05-25 at 5 44 27 PM

### CODE REVIEW

File: react/components/LowStock.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { Fragment, FunctionComponent } from 'react'
+import classNames from 'classnames'
+import styles from '../styles.css'
+
+interface Props {
+  text: string
+  threshold: number
+}
+// <div className={classNames(styles.productAvailabilityContainer, 'flex pa2')}>
```
Comment (lbebber) [inline react/components/LowStock.tsx]: :eyes:

**Review by lbebber — APPROVED** [review]

File: react/components/ProductAvailabilityWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  sellers: Array<{
+    commertialOffer: {
+      AvailableQuantity: number
+    }
+  }>
+}
+
+const ProductAvailabilityWrapper: StorefrontFunctionComponent<Props> = ({ threshold, lowStockMessage, highStockMessage }) => {
+  const valuesFromContext = useContext(ProductContext)
+  if (!valuesFromContext || isEmpty(valuesFromContext)) return null
```
Comment (lbebber) [inline react/components/ProductAvailabilityWrapper.tsx]: formatting

File: react/components/LowStock.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import classNames from 'classnames'
+import styles from '../styles.css'
+
+interface Props {
+  text: string
+  availableQuantity: number
+}
+
+const LowStock: FunctionComponent<Props> = ({ text, availableQuantity }) => {
+  const [before, after] = text.split('{quantity}')
```
Comment (lbebber) [inline react/components/LowStock.tsx:11]: perhaps check if `text` is actually a string before splitting it

**Review by lbebber — APPROVED** [review]

File: react/components/LowStock.tsx (hunk 3 above)
Comment (jgfidelis) [reply react/components/LowStock.tsx:11]: `if (isLowStock && lowStockMessage) `

already guarantees text is a string

**Pull request merged**